docs/source/get-started.md

```diff
@@ -0,0 +1,154 @@
+  }
+}
+
+dependencies {
+  compile 'com.apollographql.apollo:apollo-runtime:x.y.z'
+}
+```
+
+The plugin can then be applied as follows within your app module's `build.gradle` file:
+
+```
+apply plugin: 'com.apollographql.android'
+```
+
+> Note: The Android plugin must be applied before the Apollo plugin
+
+<h2 id="download-schema">Downloading a Schema</h2>
+
+Apollo Android requires a GraphQL schema file as input to the code generation process. A schema file is a JSON file that contains the results of an introspection query. Conventionally this file is called `schema.json`, and you store it next to your `.graphql` files.
+
+You can use `apollo-codegen` to download a GraphQL schema. This works by sending an introspection query to your server:
+
+```sh
+apollo-codegen download-schema http://localhost:8080/graphql --output schema.json
+```
+
+If needed, you can use the `header` option to add additional HTTP headers to the request. For example, to include an authentication token, use `--header "Authorization: Bearer <token>"`:
+
+```sh
+apollo-codegen download-schema https://api.github.com/graphql --output schema.json --header "Authorization: Bearer <token>"
+```
+
+> Note: In the sample project you can find the schema here => `apollo-sample/src/main/graphql/com/apollographql/apollo/sample/schema.json`.
+
+<h2 id="creating-client">Creating a Client</h2>
+
+With the installation complete and schema downloaded, let's create your Apollo Client. In most cases, you’ll want to create a single shared instance of `ApolloClient` and point it at your GraphQL server. `ApolloClient` uses `OkHttp` under the hood for handling network requests. So you will need to create an instance of the `OkHttpClient` and pass it to the `ApolloClient` builder.
+
+```java
+private static final String BASE_URL = "<your_server>/graphql";
+
+OkHttpClient okHttpClient = new OkHttpClient.Builder().build();
+
+ApolloClient apolloClient = ApolloClient.builder()
+                                        .serverUrl(BASE_URL)
+                                        .okHttpClient(okHttpClient)
+                                        .build();
+```
+
+> Note: In the sample project you can find the client here => `apollo-sample/src/main/java/com/apollographql/apollo/sample/GitHuntApplication.java`.
+
+That's it! Now your client is ready to start fetching data. 
+
+<h2 id="creating-graphql-file">Creating a .graphql File</h2>
+
+The Apollo Android plugin works by generating code that corresponds to the GraphQL queries you specify in your application's `.graphql` files. There is nothing Android specific about these queries, they can be used with other GraphQL clients as well.
+
+So if you wanted to perform the below query, you would place it in a file that ends in `.graphql`. It's common to use the query name as the file name. So in this case your file would be named, `FeedQuery.graphql`.
+
+```
+query FeedQuery($type: FeedType!, $limit: Int!) {
```

Comment:
maybe add a line like "it is important to have a name to the query or else you will fail compilation